ccm_web/server/src/app.module.ts

```diff
@@ -31,7 +31,8 @@ import { validateConfig } from './config'
         username: configService.get('db.user'),
         password: configService.get('db.password'),
         database: configService.get('db.name'),
-        models: [CanvasToken, User]
+        models: [CanvasToken, User],
+        define: { underscored: true }
```

Comment:
This may not be necessary, but seems pretty harmless, may prevent future issues.

Reply:
If/when #140 is merged, you'll need to pull those changes so you get the `logging` setting here, too.

Reply:
Whose PR was open first? 🤣

Reply:
>  define: { underscored: true }

Can this be added to `migrator.ts` as well?

Reply:
I guess I need to determine whether it's needed first, see https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/127#discussion_r658826076

Reply:
I think it matters which PR is merged first.  That is, who can click the "Merge" button faster.

On your mark! Get set! 😆 

SRSLY, it sounds like you may want to investigate some changes to this PR anyway, so the other may get merged first.  Que sera, sera.

Reply:
If that gets merged first, I'll resolve the conflict, claro que si.

Reply:
There may be some merit to including `underscored` here, as this is what determines the behavior of the `session` model automatically generated by the `connect-session-mysql` library related to `express-session`. What do you think? If we remove it, we'll have some `createdAt` type fields in `session` (the automatically generated fields seem to actually be affected by `underscored`, don't ask me why!).

Reply:
I saw in the documentation you linked to earlier that it said `created_at` would be used if `underscored` is set to true.  I think we want that.

Reply:
> So, just to make sure I understand, you've removed the object-level `underscored` definitions because it is already specified at the database level. Is that right?

Kinda. `underscored` in the models wasn't doing anything extra because `field` was specified when needed and that value overrides any other behavior. Besides, the model definition is separate from the migration, they need to be kept in sync in our current setup. If we wanted to try to use `underscored`, I think the important place to set this would be in `migrator.ts`. However, I wasn't seeing the `underscored` naming strategy being used consistently when I tried it there. So I'm relying on `field` instead. I hope that helps explain. This is confusing to me as well. I recommend trying it out yourself.

Reply:
I'm keeping `underscored` here because, as I said, this `sequelize` instead is responsible for the `session` table via `sync` (see `main.ts`). If you remove this setting, you should see that the session table uses `createdAt`. I am currently keeping this setting so that it uses `created_at`.